README.md

```diff
@@ -1,5 +1,6 @@
 Windows 10 IoT Core for NXP i.MX Processors
 ==============
+<PUT PIPELINE STATUS BADGE HERE!>
```

Comment:
Leave this part out. We'll add it once the pipeline is up and running.

Reply:
Talked with Chris, we should be able to replace this with the badge as soon as the pull request is done. Leaving it in.